releasenotes/notes/support_backend_v2-9eeb4690199b01d1.yaml

```diff
@@ -0,0 +1,4 @@
+---
+features:
+  - |
+    Update Aer Backend to BackendV2. Refer to `#1681<https://github.com/Qiskit/qiskit-aer/issues/1681>`.
```

Comment:
This needs a bit more detail to document the user upgrade implications. The only one that I can think of is `backend.name` is now a string attribute not a method.

Reply:
Add the detailed description to the release note.